webapp/src/main/java/org/apache/atlas/notification/EntityNotificationListenerV2.java

```diff
@@ -98,7 +100,7 @@ public void onClassificationsAdded(AtlasEntity entity, List<AtlasClassification>
 
     @Override
     public void onClassificationsAdded(List<AtlasEntity> entities, List<AtlasClassification> classifications) throws AtlasBaseException {
-        notifyClassificationEvents(entities, CLASSIFICATION_ADD, classifications);
+        notifyClassificationEvents(entities, CLASSIFICATION_ADD, classifications, true);
```

Comment:
This won't just send inline for classification prop, but whenever classification gets added. Want to discuss this

Reply:
Let's do this only for tasks